docs/adr/0001-update-major-version-with-uswds-minor-version.md

```diff
@@ -0,0 +1,100 @@
+
+Therefore, any time we decide to update the version of USWDS used in ReactUSWDS, we should expect to perform the following changes:
+
+- Update the USWDS version listed in both `devDependencies` and `peerDependencies`
+- Perform any API or implementation changes to existing components detailed in the USWDS release notes
+  - For example, from [2.10.1](https://github.com/uswds/uswds/releases/tag/v2.10.1):
+    > Use proper semantic markup for footer title: Now we're using a p instead of an h3 for usa-footer\_\_logo-heading since the name of the project or agency is hot a heading in this context.
+
+The questions this ADR seeks to answer are:
+
+- When performing a USWDS version update as described above, what kind of release should that result in for ReactUSWDS? (i.e., major / minor / patch)
+  - At minimum, this update will always require users to also update their version of USWDS.
+- Should USWDS version updates be timed and released in sequence with the components they implement?
+  - For example, should we aim to release all components introduced in USWDS 2.9.0 before releasing the update to 2.10.0 and any of its components?
+
+For reference, USWDS does mention semantic versioning in [their own release process documentation](https://github.com/uswds/uswds/wiki/Release-process#versioning), and documents breaking changes (constituting major releases) as follows:
+
+> In most software projects, the "public API" corresponds to a single set of programming constructs, such as public classes or functions. Because the USWDS consists of tightly-bound HTML, CSS, and JavaScript, we must consider any "breaking" change to any of these as a change to the public API. For example, any of the following should trigger a major version increment:
+>
+> - Changing the name of any .usa- class name (documented or not)
+> - Changing the way in which elements with .usa- class names are structured in HTML
+> - Changing the HTML "API" for any of our interactive components, such as the accordion
+
+## Decision Drivers
+
+- We want to provide as many USWDS components as possible to the widest audience as possible, taking into account users who may be version-locked to a legacy USWDS version for whatever reason
+- We don't want to impede development and release of USWDS components, and we aim to "catch up" to the current USWDS release eventually
+
+## Considered Options
+
+- Bump the ReactUSWDS major version when updating USWDS minor & major versions
+- Bump the ReactUSWDS version with the same release type as USWDS
```

Comment:
I vote for `Bump the ReactUSWDS version with the same release type as USWDS`. We probably should also acknowledge this means we don't anticipate maintaining multiple active versions of the library. For example, we wouldn't back port a new component or bug fix to a previous USWDS version.